subsys/bluetooth/mesh/sensor_cli.c

```diff
@@ -937,8 +944,14 @@ int bt_mesh_sensor_cli_series_entry_get(
 	net_buf_simple_add_le16(&msg, sensor->id);
 
 	col_format = bt_mesh_sensor_column_format_get(sensor);
-	if (col_format == NULL) {
-		net_buf_simple_add_le16(&msg, column->start.val1);
+	if (!col_format) {
+		if (column->start.val1 < 0xFF) {
+			net_buf_simple_add_u8(&msg, column->start.val1);
+		} else if (column->start.val1 < 0xFFFF) {
+			net_buf_simple_add_le16(&msg, column->start.val1);
+		} else {
+			net_buf_simple_add_le24(&msg, column->start.val1);
+		}
```

Comment:
MshMDLd1.1r07, 4.2.15 Sensor Column Get (emphasis mine):
> [...] For the device properties that have one or two fields in the Property Value, the Raw Value A field identifies a **2-octet starting index** of the measured value counting backwards from the most recent value (represented as index 0x0000).

Reply:
Ok, so this is the same as above, with which I totally agree. Seems like PTS has bug and excepts Raw Value X to be 1 octet long, like for properties with 3 or more fields. I'll file an issue for that, and remove this from PR.